src/Symfony/Component/Routing/Requirement/EnumRequirement.php

```diff
@@ -15,36 +15,42 @@
 
 final class EnumRequirement implements \Stringable
 {
-    /**
-     * @var string[]
-     */
-    private readonly array $values;
+    private string $requirement;
```

Comment:
@nicolas-grekas Why did you remove the readonly?

Reply:
Not intentional! (But read-only on private is not useful either so 🤷‍♂️ 😆)